llmfoundry/eval/datasets/in_context_learning_evaluation.py

```diff
@@ -1423,6 +1423,7 @@ def build_icl_dataloader(
             example_delimiter=example_delimiter,
             continuation_delimiter=continuation_delimiter,
             destination_path=destination_path,
+            prelimiter=prelimiter,
```

Comment:
The `prelimiter` was not passed into several of the ICL task datasets. For more details, see my PR in composer [here](https://github.com/mosaicml/composer/pull/3069).